andela/travel_tool_front
# #161376806 Remove dummy approver data on request modal

#### What does this PR do?
* Replaces dummy approver information on request modal for approved request

#### Description of Task to be completed?
- replace dummy approver data on request modal with approvers image and name
- fix checkbox style issue on new request form

#### How should this be manually tested?
- Clone this repo - git clone https://github.com/andela/travel_tool_front.git
- Switch to the repo directory - cd travel_tool_front
- Check out this branch - git checkout ft-update-dummy-approver-data-161376806
- Install dependencies - yarn install
- Start the server - make start
- Set up the server by following this guide
- Navigate to localhost:3000 in a browser
- Create a new request and add yourself as manager
- Navigate to `My Approvals` page and approve the just created request
- Click on the approved request, you should see the details of the approver in this format
     ` [image] [approver name] [message] [time]`

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#161376806](https://www.pivotaltracker.com/story/show/161376806)

#### Screenshots (if appropriate)
<img width="1440" alt="screen shot 2018-10-24 at 9 17 11 am" src="https://user-images.githubusercontent.com/23608060/47416991-d97ca880-d76e-11e8-8d89-779ef0ceaef5.png">

#### Questions:
